source/usage-examples/findOne.go

```diff
@@ -0,0 +1,54 @@
+//Package main will provides a code example showing
+//how to find a document using the MongoDB Go Driver.
```

Comment:
Thank you for the note. 

After [reviewing](https://github.com/golang/go/blob/master/src/syscall/mkasm.go) [some](https://github.com/golang/go/blob/master/src/syscall/mksyscall_windows.go) [of](https://github.com/golang/go/blob/master/src/os/exec/read3.go) the package doc comment for main packages in the [golang repo](https://github.com/golang/go), it seems like it is appropriate to reference the go file in a docstring as "findOne.go finds a document...", "this program finds a document...", or just leaving off a package doc comment. Do you have a preference for which one? For the sake of picking one, I am picking "this program ..." as I don't think readers will know the filename of the code example.

Main functions seem to not get a doc comment, so I am deleting the doc comment I have on my main function. 

Fix:

Reply:
I think there may be some disconnect in communication after reading your response, so I'll reword my initial comments.

The grammar in the comment is incorrect. "X will provides" does not have subject-verb agreement in terms of plurality. The sentence also uses future tense which we should avoid according to the style guide.

The second comment refers to how you're using "package main". The suggestion is to replace it with "This go file includes a code example that shows..." At current, my opinion is to omit the entire code comment because it's redundant. For situations in which you must include a code comment (cannot think of any offhand) similar to this, you could just call it a code example and not something that includes a code example.

Reply:
Thanks for following up. I have removed the package comment. I included it as I thought it was good go style to include a package comment, but I can see how in the context of this example it is not helpful to readers. 

Fix: https://github.com/mongodb/docs-golang/pull/2/commits/66e281edfc8e36176f3a46e14be09d38b9a2631a